docker-compose.yml

```
@@ -34,3 +37,18 @@ services:
       options:
         max-size: "10m"
         max-file: "3"
+    networks:
+      openhim:
+      kafka:
+      elastic:
+
+networks:
+  openhim:
+    name: openhim_public
+    external: true
+  kafka:
+    name: kafka_public
+    external: true
+  elastic:
+    name: elastic_public
+    external: true
```

```suggestion
    external: true

```

add new line at end of file